src/mapping/mod.rs

```diff
@@ -1,4 +1,6 @@
-use crate::event::{Event, Value};
+#![allow(deprecated)] // TODO: This uses things deprecated in https://github.com/timberio/vector/pull/5374
```

Comment:
I am under the impression all this code is going away and so all the marked deprecated code can when that does.